cpp/src/parquet/properties.h

```diff
@@ -541,6 +544,8 @@ class PARQUET_EXPORT WriterProperties {
     }
 
     /// Enable writing page index for column specified by `path`. Default disabled.
+    /// Note that it does not write statistics to the page header once page index is
+    /// enabled.
```

Comment:
Update this comment as well? Or just removed it as things are explained in more detail above.

Reply:
Sorry for missing that! Fixed.